QuestionBot/QuestionBot.UnitTests/Model/QuestionMessageListenerTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
```

Comment:
These tests are missing at least 2 common error cases and one of the missing cases will cause a runtime error. Don't forget to test all your error cases! :)